src/components/Quote.vue

```diff
@@ -221,10 +187,15 @@ export default {
 		}
 		&__text {
 			color: var(--color-text-light);
+			white-space: pre-wrap;
+			word-break: break-word;
 			& p {
-				max-width: 662px;
 				text-overflow: ellipsis;
 				overflow: hidden;
+				// Allow 2 lines max and ellipsize the overflow;
```

Comment:
clamping is experimental, be aware of the browsers talk claim to support (no idea, just a warning :wink: )

Reply:
The latest version of every browser but ie supports it. We recommend the latest version in the talk page. For ie, the substring() method above is a nice fallback!